src/khoj/routers/helpers.py

```diff
@@ -927,11 +928,13 @@ async def text_to_image(
                 response = state.openai_client.images.generate(
                     prompt=improved_image_prompt,
                     model=text2image_model,
+                    style="vivid",
                     response_format="b64_json",
                     extra_headers=auth_header,
                 )
                 image = response.data[0].b64_json
-                decoded_image = base64.b64decode(image)
+                # Decode base64 png and convert it to webp for faster loading
+                webp_image_bytes = convert_image_to_webp(base64.b64decode(image))
```

Comment:
Let's move this `text_to_image` function into a separate file? Maybe we can create a `/processor/image` folder and make `generation.py` over there with this function. Ideally the per-model implementations should also be broken down further, e.g., a separate function for each OpenAI, Replicate, and StabilityAI. IMO it'll make the code more readable/organized.